index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE HTML>
+<html lang="en">
+  <head>
+    <meta charset=“UTF-8“ />
+    <title></title>
+    <link rel="stylesheet" href="styles/styles.css" />
+    <link href="https://fonts.googleapis.com/css?family=Anton|Questrial|Roboto" rel="stylesheet">
+
+
+
+  </head>
+  <body class="firmware-body">
+
+
+      <h1 class="home-title" id='title'>WELCOME TO FIRMWARE</h1>
+      <h2 class="home-title" id='subtitle'>THE BIGGEST SERVICE FINDER ONLINE</h2>
+
+      <nav class='navigation'>
+        <input type="checkbox" id="navToggle" />
```

Comment:
You should be consistent with the naming strategy you use for ids and classes. Either use camelCase, dashes or underscore but choose one and use it everywhere. In this case, you should go with the dash, as you did for classes.